css/properties/position.json

```diff
@@ -210,6 +210,54 @@
               "deprecated": false
             }
           }
+        },
+        "position_sticky_table_elements": {
+          "__compat": {
+            "description": "Table elements as <code>sticky</code> positioning containers",
+            "support": {
+              "webview_android": {
+                "version_added": "64"
+              },
+              "chrome": {
+                "version_added": "64"
```

Comment:
According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.